packages/bitcore-wallet-client/src/lib/api.ts

```diff
@@ -1268,8 +1272,13 @@ export class API extends EventEmitter {
 
     var args = this._getCreateTxProposalArgs(opts);
 
+    if (opts.coin === 'bch' && opts.network === 'testnet') {
```

Comment:
Can't be use always /v3/proposals? and just post a different signingMethod on the proposal
object?